lib/sinon/proxy-call.js

```diff
@@ -220,7 +220,7 @@ var callProto = {
             }
         }
         if (this.stack) {
-            // Omit the error message and the two top stack frames in sinon itself:
+            // Emit the error message and the two top stack frames in sinon itself:
```

Comment:
Okay, yes upon investigation you're totally right here, thanks, I misunderstood what was going on here. I read "in sinon itself" as "add the stack frame to sinon's output" when I now realize I think what it meant is "omit the top two stack frames, which will be in sinon itself".

After re-orienting myself, I do think it's a little confusing because if this code wasn't here, the stack frames wouldn't show up at all, and also because we're actually just pulling out only the third stack frame, rather than stripping off the first two and leaving the rest, right?

So the comment as it was is certainly better than my replacement, but what about something like:
```
          if (this.stack) {
            // If present, add the third stack frame to the error output for context
            // Skip the first two stack frames, which will be within Sinon code
            callStr += (this.stack.split("\n")[3] || "unknown").replace(
                /^\s*(?:at\s+|@)?/,
                " at "
            );
          }
```
(Code is the same, I just left it there for context)

Reply:
Definitely an improvement!